splash/network_manager.py

```diff
@@ -86,6 +153,7 @@ class ProxiedQNetworkAccessManager(QNetworkAccessManager):
     * Tracks information about requests/responses and stores it in HAR format,
       including request and response content.
     * Allows to set per-request timeouts.
+    * Handles per-request response size limits.
```

Comment:
I don't find how to set the "per-request response size limit" in this PR.

Reply:
By adding `max_size` to the render options of a request.

If you mean that the documentation should explain this, indeed.